File: translate-copilot/loops.cpp
// C++ version
#include <iostream>
#include <vector>
#include <string>

int main() {
    std::vector<std::string> fruits = {"apple", "banana", "pear"};
    for (const auto& fruit : fruits) {
        std::cout << "I like " << fruit << std::endl;
    }
    return 0;
}

File: translate-copilot/loops.cs
// C# version
using System;
using System.Collections.Generic;

class Loops {
    static void Main() {
        List<string> fruits = new List<string> { "apple", "banana", "pear" };
        foreach (string fruit in fruits) {
            Console.WriteLine("I like " + fruit);
        }
    }
}
